lib/AST/Attr.cpp

```diff
@@ -261,10 +261,16 @@ static void printShortFormAvailable(ArrayRef<const DeclAttribute *> Attrs,
   Printer << "@available(";
   auto FirstAvail = cast<AvailableAttr>(Attrs.front());
   if (Attrs.size() == 1 &&
-      FirstAvail->isLanguageVersionSpecific()) {
+      (FirstAvail->isLanguageVersionSpecific() || 
+	   FirstAvail->isSwiftPMManifestVersionSpecific())) {
```

Comment:
This can just be "is it platform agnostic", since we already know there's a version number.